js/templates/buyWizard.html

```diff
@@ -28,10 +28,12 @@ <h3 class="custCol-text padding30"><%= polyglot.t('CheckExchanges') %> </h3>
         </div>
         <div class="txt-bright pad20">
           <div class="txt-center row10 textSize20px custCol-text">
-            <%= polyglot.t('buyFlow.SendBTCtoAddress', {amount: '<span class="js-buyWizardDetailsTotalBTC"></span>'}) %>
-          </div>
-          <div class="txt-center row10">
-            <a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>
+            <%=
+              polyglot.t('buyFlow.SendBTCtoAddress', {
+                amount: '<span class="js-buyWizardDetailsTotalBTC"></span>',
+                recipient: '<a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>'
```

Comment:
I adjusted it to be an inline element within the same container as the rest of the sentence. If they place it in the middle of the sentence it will fall inline into place. If it's a guid (which is a large string), it will take up it's own line. If it's a handle (which is short), it will fall nicely inline in place. I played around with it in the inspector with various values and it seemed to look ok.

Is it critical that it needs to be a block level element below the sentence? If so, then I should probably just revert this particular change.